Task-based edit SAML SSO Setup (#5993)
braze-inc/braze-docs: _docs/_user_guide/administrative/access_braze/single_sign_on/set_up.md

## SAML SSO setup

> ### Step 1. Configure your identity provider

Reviewer: ```suggestion
### Step 1: Configure your identity provider
```

Braze only requires `email` in the SAML Assertion.
{% endalert %}

> ### Step 2. Configure Braze

Reviewer: ```suggestion
### Step 2: Configure Braze
```

When you finish setting up Braze in your IdP, your IdP will provide you with a Target URL and `x.509` certificate to input into your Braze account. 

After your account manager turns on SAML SSO for your account, go to **Settings** > **Admin Settings** > **Security Settings** and toggle the SAML SSO section to **ON**. 

{% alert note %}
If you are using the [older navigation]({{site.baseurl}}/navigation), select your account icon and go to **Company Settings** > **Security Settings** to find the SAML SSO section.
{% endalert %}

On the same page, input the following:

| Requirement | Details |
|---|---|

> | `SAML Name` | This will appear as the button text on the login screen.<br>This is typically your IdP name, like "Okta." |

Reviewer: ```suggestion
| `SAML Name` | This will appear as the button text on the login screen.<br>This is typically your identity provider's name, like "Okta." |
```

## Restriction

> You can restrict the members of your organization to sign-in with either Google SSO or SAML SSO. To turn on restrictions, go to **Security Settings** and select either **Enforce Google SSO only login** or **Enforce custom SAML SSO only login**.

Reviewer: ```suggestion
You can restrict the members of your organization to only sign in with either Google SSO or SAML SSO. To turn on restrictions, go to **Security Settings** and select either **Enforce Google SSO only login** or **Enforce custom SAML SSO only login**.
```
